lib/msal-common/src/request/ScopeSet.ts

```diff
@@ -21,29 +21,49 @@ export class ScopeSet {
     // Boolean denoting whether scopes are required. Usually used for validation.
     private scopesRequired: boolean;
 
-    constructor(inputScopes: Array<string>, appClientId: string, scopesRequired: boolean) {
+    constructor(
+        inputScopes: Array<string>,
+        appClientId: string,
+        scopesRequired: boolean,
+        nonSPA?: boolean
```

Comment:
This is not reflective of how it is used for the msal browser use case, I'm not sure I like the name change here.

Reply:
This class has to be rewritten to fit a more generic rhetoric. I added this as a temporary mitigation. Can create a new PR to close this in a cleaner fashion.